src/towncrier/_settings/load.py

```diff
@@ -164,11 +182,13 @@ def parse_toml(base_path: str, config: Mapping[str, Any]) -> Config:
             failing_option="template",
         )
 
+    filename = config.get("filename", "NEWS.rst")
+    is_md = filename.endswith(".md")
```

Comment:
Can we have all the `if_md` values  and default selection  in a common block.

Is ok to even move this  to a separate function like `defaults = _get_missing_defaults(config)"


```suggestion
    underlines = config.get("underlines", "")
    title_prefixes=config.get("title_prefixes", "")
    if filename.endswith(".md"):
        # Try to use markdown default.
        if no underlines:
            underlliens = ["", "", ""]

        if no underlines:
            # Fallback to rst defaults.
            underlines = ["=", "-", "~"]
```

And I hope we don't need md_prefixes and md_bullet, as we can handle all this in the custom `tempalates/default.md` 

What do you think?